master/internal/core_experiment.go

```diff
@@ -388,6 +389,11 @@ func (m *Master) parseCreateExperiment(ctx context.Context, req *apiv1.CreateExp
 
 	taskSpec.Project = config.Project()
 	taskSpec.Workspace = config.Workspace()
+	if p.Id > 1 {
```

Comment:
We are changing ```Project/Workspace``` which is used in Slurm. Is this change of this going to affect any behavior in Slurm? Is this field not being set always causing any bugs in Slurm? 

I know I said it was a bug but it is probably worth checking that this does not affect slurm.